packages/rule-components/src/ReportDetails/index.scss

```diff
@@ -1,13 +1,68 @@
+@import '~@redhat-cloud-services/frontend-components-utilities/files/Utilities/_variables';
+
 .ins-c-rule__report-detail {
-  padding: var(--pf-global--gutter--md) var(--pf-global--gutter);
+  padding: var(--pf-global--gutter--md) 40px;
 
   .ins-c-rule__report-detail-link {
     margin-bottom: var(--pf-global--spacer--xs);
     margin-left: var(--pf-global--spacer--form-element);
   }
 
-  .ins-c-rule__risk-detail {
+  .ins-c-rule__risk-detail-title {
     margin-bottom: var(--pf-global--spacer--sm);
+    font-weight: var(--pf-global--FontWeight--light);
+  }
+
+  .ins-c-rule__risk-detail-description-text {
+    color: var(--pf-global--Color--200);
+    size: var(--pf-chart-global--FontSize--sm);
+  }
+
+  .ins-c-rule__link-in-description {
+    display: block;
+    margin-top: var(--pf-l-stack--m-gutter--MarginBottom);
+  }
+
+  .ins-c-rule__risk-description {
+    .ins-c-rule__battery .label {
+      font-weight: bold;
+    }
+
+    .ins-c-rule__severity-level-1 .label {
+      color: $ins-color--gray;
+    }
+
+    .ins-c-rule__severity-level-2 .label {
+      color: $ins-color--medium;
+    }
+
+    .ins-c-rule__severity-level-3 .label {
+      color: $ins-color--high;
+    }
+
+    .ins-c-rule__severity-level-4 .label {
+      color: $ins-color--critical;
+    }
+  }
+
+  .ins-risk-of-change-icon {
+    color: $ins-color--high;
+    font-size: 28px;
```

Comment:
This should be `var(--pf-global--FontSize--2xl)`

Reply:
it's still 24px, not 28px as I need